.github/workflows/publish.yaml

```diff
@@ -0,0 +1,58 @@
+name: Publish to PyPi and create GitHub Release
+on:
```

Comment:
It didn't run on this PR, would be nice if we could run it after all tests have passed

Reply:
I read on this, and new workflow files must be on the `main`/`master`/`default` branch in order to trigger them.

Reply:
oki, then lets merge this!